Testing image embed

jordynmay opened the pull request.

jordynmay committed:
Added image url

cathyrae903 commented:
The image still didn't show up with this URL, but I think I know why. If I go to the URL in the browser and right click on the image -> View Image and then copy and paste _that_ URL into the JSON file, it works. I believe that's the case because the imgur URL leads to a webpage rather than an image (and doesn't have an extension for an image). 

Could @jordynmay test this and see if you get the same results? If you do, could you then upload the steps to add the image to imgur? You'll want to put these steps in the readme.md file and replace the current description (second paragraph) that isn't working.

cathyrae903 committed:
Update image URL to reflect competition's changes

Successful changes were made to `questions.json` in 2021 competition. This is updating the sample `questions.json` file to reflect these changes.

cathyrae903 reviewed (approved):
@jordynmay I synced your changes and everything works perfectly!
